samples/spring-valueentity-customer-registry/src/main/java/com/example/api/CustomerSummaryByName.java

```diff
@@ -0,0 +1,26 @@
+package com.example.api;
+
+import com.example.domain.Customer;
+import kalix.javasdk.view.View;
+import kalix.springsdk.annotations.Query;
+import kalix.springsdk.annotations.Subscribe;
+import kalix.springsdk.annotations.Table;
+import kalix.springsdk.annotations.ViewId;
+import org.springframework.web.bind.annotation.GetMapping;
+import reactor.core.publisher.Flux;
+
+@ViewId("view_summary_customers_by_name")
+@Table("customers")
+// tag::class[]
+@Subscribe.ValueEntity(CustomerEntity.class)
+public class CustomerSummaryByName extends View<Customer> { // <1>
+
+  @GetMapping("/summary/by_name/{customerName}")   // <2>
+  @Query(
+      value = "SELECT customerId AS id, name FROM customers WHERE name = :customerName", // <3>
+      streamUpdates = true) // <4>
+  public Flux<CustomerSummary> getCustomer() { // <5>
+    return null;
+  }
```

Comment:
This is a good example to illustrate relational projection, but the original ticket was about doing a transformation and indexing the transformed model. 

The effect is slightly different. 
Here, we are persisting `Customer` to the table with a index on name. When we retrieve the results, we map it to `CustomerSummary`. 

The other alternative is to have a `onUpdate` method that receives `Customer` and returns `CustomerSummary`. 

For that, we need an example with `@Subscribe.ValueEntity(CustomerEntity.class)` at method level, not at type level.

Reply:
yeah, I see now.. I took over the issue but thought it was about that section 😂 

Anyway, I will see where to fit that new example.. this is one we didn't have in Java for same reason, right?